tools/twix/src/panels/behavior_simulator.rs

```diff
@@ -123,7 +123,7 @@ impl Widget for &mut BehaviorSimulatorPanel {
         if self.playing {
             let now = ui.input(|input| input.time);
             let time_elapsed = now - self.playing_start;
-            new_frame = Some((time_elapsed * 83.0) as usize);
+            new_frame = Some((time_elapsed * 83.0 * 5.0) as usize);
```

Comment:
I changed the way playback is done completely and added a slider for playback speed. The new implementation also allows for negative speeds and wrapping in both directions.
Playback speed still defaults to 5x because 1x is painfully slow.